### PR TITLE
Console 67 self service license trials and graceful license expiry merge

### PR DESCRIPTION
**Pull request by jvorcak**

### CODE REVIEW

File: proto/gen/openapi/openapi.yaml
Comment (malinskibeniamin) [inline proto/gen/openapi/openapi.yaml:1]: not sure if it needs to generate a new `openapi.json` file

File: frontend/src/components/license/LicenseNotification.tsx [hunk 1]
```diff
@@ -76,16 +91,9 @@ export const LicenseNotification = observer(() => {
...
-            {soonToExpireLicenses.length > 0 && (
-              <Button variant="outline" size="sm" as="a" target="_blank" href="https://redpanda.com/license-request">
-                Renew license
-              </Button>
-            )}
-            {showEnterpriseFeaturesWarning && (
-              <Button variant="outline" size="sm" as="a" target="_blank" href="https://www.redpanda.com/try-redpanda">
-                Request a trial
-              </Button>
-            )}
```
Comment (malinskibeniamin) [inline frontend/src/components/license/LicenseNotification.tsx:88]: Do we want to keep this or is it intentional to request license?

Comment (malinskibeniamin) [reply frontend/src/components/license/LicenseNotification.tsx:88]: Should we use https://redpanda.com/license-request?

**Review by malinskibeniamin — APPROVED** [review]
LGTM, some minor comments

File: frontend/src/components/license/LicenseNotification.tsx (hunk 1 above)
Comment (jvorcak) [reply frontend/src/components/license/LicenseNotification.tsx:88]: I believe this was an explicit request on one of the last calls.

**Pull request merged**